app-misc/vcontrold/vcontrold-0.98.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+DEPEND="dev-libs/libxml2
+	man? ( dev-python/docutils )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -e "s/@VERSION@/${PV}/" "${WORKDIR}/${P}/src/version.h.in" \
+		> "${WORKDIR}/${P}/src/version.h" || die "Setting version failed"
```

Comment:
You can drop the -e (sed assumes the first argument is to be executed), and rather than redirect the output, edit in place with -i.

Plus, you should be  able to drop `"${WORKDIR}/${P}/`.

Reply:
I thought it would be better to redirect the output, as we have `version.h.in` to be edited and `version.h` to be created – otherwise, I would have to do `cp version.h.in version.h` before, wouldn't I?